acceptance-test/acceptance_test.go

```diff
@@ -39,40 +39,35 @@ var suite = []FeatureFile{
 }
 
 func TestFeatures(t *testing.T) {
-	var mng *Mongo
-	var flyte *Flyte
-	var err error
-	defer tearDown(flyte, mng)
-
-	mng, err = StartMongo()
+	mng, err := StartMongo()
```

Comment:
Not really related with the ticket, but this test was never shutting down both containers. `flyte` and `mng` values in the defer block points to a different var values.